src/modules/ie-objects/controllers/ie-objects.controller.ts

```diff
@@ -441,4 +441,51 @@ export class IeObjectsController {
 
 		return limitedObject;
 	}
+
+	@Get()
+	public async getIeObjectByIds(
+		@Query('id') ids: string[],
+		@Headers('referer') referer: string,
+		@Req() request: Request,
+		@SessionUser() user: SessionUserEntity
+	): Promise<IeObject[] | Partial<IeObject>[]> {
+		const ieObject: IeObject[] = await this.ieObjectsService.findBySchemaIdentifier(
+			ids,
+			referer,
+			getIpFromRequest(request)
+		);
+
+		const visitorSpaceAccessInfo =
+			await this.ieObjectsService.getVisitorSpaceAccessInfoFromUser(user);
+
+		return ieObject.map((ieObject) => {
```

Comment:
don't use variable names in for loops or map functions that already exist outside of the loop